docs/core/whats-new/dotnet-core-3-0.md

```diff
@@ -9,16 +9,17 @@ ms.author: adegeo
 ms.date: 07/25/2019
```

Comment:
```suggestion
ms.date: 08/21/2019
```

Reply:
I dont know why I forget the date ALL THE TIME.. hahahaha